helm/odigos/values.yaml

```diff
@@ -5,7 +5,7 @@ imagePullSecrets: []
 # By default, images are pulled from odigos registry at `registry.odigos.io`
 # If you use custom or internal registry to serve in your cluster, you can set the imagePrefix to your registry.
 # For example, if you set imagePrefix to `myregistry.io/odigos`, the images will be pulled from `myregistry.io/odigos/odigos-<component>:<tag>`
-imagePrefix:
+imagePrefix: 'registry.odigos.io'
```

Comment:
didn't try to run it myself, but I think if we populate the imagePrefix in the `values.yaml`, then we can conflict with the logic for openshift [here](https://github.com/odigos-io/odigos/blob/de1554d717c8a0d83049007f9d65c6057c843a84/helm/odigos/templates/_helpers.tpl#L4).